analysis/data_utils.py

```diff
@@ -106,15 +106,17 @@ def filter_max_time(experiment_df, max_time):
     return experiment_df[experiment_df['time'] <= max_time]
 
 
-def add_bugs_covered_column(benchmark_df):
-    """Return a modified benchmark df in which the |crash_key| column is
-    replaced with a |bugs_covered| column."""
-    new_df = benchmark_df.drop(columns='crash_key').drop_duplicates()
-    new_df['bugs_covered'] = new_df.apply(lambda x: benchmark_df[
-        (benchmark_df['trial_id'] == x['trial_id']) &
-        (benchmark_df['fuzzer'] == x['fuzzer']) &
-        (benchmark_df['time'] <= x['time'])]['crash_key'].nunique(),
-                                          axis=1)
+def add_bugs_covered_column(experiment_df):
```

Comment:
Beautiful! Thanks so much!

Please consider adding a unit test for this if you happen to have time (otherwise no worries).

Reply:
added a simple unit test!